ptrace.c

```diff
@@ -36,6 +36,12 @@
 
 #include "ptrace.h"
 
+#if defined(__GLIBC__)
+#  define PTRACE_REQUEST_TYPE enum __ptrace_request
+#else
```

Comment:
If the host is not using glibc, so which libc is in use?

Reply:
musl libc, uClibc, anything that's not specifically gnu libc